src/validator/CreditCardVendor.java

```diff
@@ -0,0 +1,32 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardVendor {
+	private String name;
+	private ArrayList<Integer> numberOfChars;
```

Comment:
I think this name is unfortunate. `allowedCardNumberLengths` is probably much better because it tells the user what this field actually stores. Also, please don't use implementations on the left side - use the most general interface, I think in this case it should be `List` or even better `Collection`, because it's a good idea to use `Set` here